src/coreComponents/physicsSolvers/multiphysics/poromechanicsKernels/SinglePhasePoromechanics.hpp

```diff
@@ -300,7 +300,7 @@ class SinglePhaseBulkDensityKernel
                 localIndex const q ) const
   {
     m_bulkDensity[ei][q] =
-      ( 1 - m_porosity[ei][q] ) * m_rockDensity[ei][q] + m_porosity[ei][q] * m_fluidDensity[ei][q];
+      ( 1 - m_porosity[ei][q] ) * m_rockDensity[ei][q] + m_porosity[ei][q] * m_fluidDensity[ei][0];
```

Comment:
Maybe we should have a double loop? If we ever change the `maxNumQuadPoints` this will break.